CHANGELOG.md

```diff
@@ -95,6 +95,11 @@
   [Frederick Pietschmann](https://github.com/fredpi)
   [#2670](https://github.com/realm/SwiftLint/issues/2670)
 
+* Fix false positives on `unused_capture_list` which resulted in an array
```

Comment:
Changelog entry isn't necessary because this rule hasn't shipped in a release yet.